files/usr/share/cinnamon/applets/menu@cinnamon.org/applet.js

```diff
@@ -1174,6 +1174,10 @@ class CinnamonMenuApplet extends Applet.TextIconApplet {
         this.menu = new Menu(this, orientation);
         this.menuManager.addMenu(this.menu);
 
+        const edit_item = new PopupMenu.PopupIconMenuItem("Menu editor", "document-edit", St.IconType.SYMBOLIC);
```

Comment:
Change to "Edit menu" please, and it needs to be translated, so wrap it like this: `_("Edit menu")` (like line 1172 above)